### PR TITLE
pydeck: Publish 0.8.0b1

### PR DESCRIPTION
**Pull request by Jesus89**
#### Change List
- Bump version (using deck.gl 8.8)
- Minor improvements in docs and examples
- Update ipywidgets version range (to make it compatible with jupyter-widgets/base ^4) [More info](https://ipywidgets.readthedocs.io/en/latest/migration_guides.html#migrating-from-7-x-to-8-0)

#### Release
- #7026
- #7125

### CODE REVIEW

File: bindings/pydeck/examples/02 - Scatterplots.ipynb [hunk 1]
```diff
@@ -121,7 +121,7 @@
    },
    "outputs": [],
    "source": [
-    "from IPython.core.display import display\n",
```
Comment (kylebarron) [inline bindings/pydeck/examples/02 - Scatterplots.ipynb:124]: `display` is also exported from the top-level `IPython` 🤷‍♂️

File: modules/jupyter-widget/package.json [hunk 2]
```diff
@@ -29,13 +29,13 @@
...
   "dependencies": {
-    "@deck.gl/aggregation-layers": "8.8.0-alpha.6",
-    "@deck.gl/carto": "8.8.0-alpha.6",
-    "@deck.gl/geo-layers": "8.8.0-alpha.6",
-    "@deck.gl/google-maps": "8.8.0-alpha.6",
-    "@deck.gl/json": "8.8.0-alpha.6",
-    "@deck.gl/layers": "8.8.0-alpha.6",
-    "@deck.gl/mesh-layers": "8.8.0-alpha.6",
+    "@deck.gl/aggregation-layers": "^8.8.0",
+    "@deck.gl/extensions": "^8.8.0",
```
Comment (kylebarron) [inline modules/jupyter-widget/package.json:33]: Why was this added?

File: bindings/pydeck/examples/02 - Scatterplots.ipynb (hunk 1 above)
Comment (Jesus89) [reply bindings/pydeck/examples/02 - Scatterplots.ipynb:124]: I followed the warning to apply the update. I think `display` in `IPython` is the namespace, not the function.

File: modules/jupyter-widget/package.json (hunk 2 above)
Comment (Jesus89) [reply modules/jupyter-widget/package.json:33]: The package `@deck.gl/extensions` is a new peerDependency of `@deck.gl/carto` (https://github.com/visgl/deck.gl/blob/master/modules/carto/package.json#L59). I thought it would be interesting to have it for all the generic available in `@deck.gl/jupyter-widget`. 

However, if we don't want to add it for all, there is an alternative to be applied in `pydeck-carto` to load it dynamically for the custom layer using a combined load:

```
https://cdn.jsdelivr.net/combine/npm/@deck.gl/extensions@~8.8.*/dist.min.js,npm/@deck.gl/carto@~8.8.*/dist.min.js
```

Comment (Jesus89) [reply modules/jupyter-widget/package.json]: I finally removed this library and use the second method, which is more flexible. It can be added in the future if needed by a core module.

**Comment by ajduberstein** [comment]
Looks good to me, though I think you need to update the ``_version.py`` file

**Comment by Jesus89** [comment]
@ajduberstein the _version file was updated in this (merged) PR: https://github.com/visgl/deck.gl/pull/7125

**Review by ajduberstein — APPROVED** [review]

**Comment by Jesus89** [comment]
Pydeck 0.8.0b1 released on pypi: https://pypi.org/project/pydeck/0.8.0b1/

File: modules/jupyter-widget/package.json [hunk 3]
```diff
@@ -29,13 +29,12 @@
...
   },
   "dependencies": {
-    "@deck.gl/aggregation-layers": "8.8.0-alpha.6",
-    "@deck.gl/carto": "8.8.0-alpha.6",
-    "@deck.gl/geo-layers": "8.8.0-alpha.6",
-    "@deck.gl/google-maps": "8.8.0-alpha.6",
-    "@deck.gl/json": "8.8.0-alpha.6",
-    "@deck.gl/layers": "8.8.0-alpha.6",
-    "@deck.gl/mesh-layers": "8.8.0-alpha.6",
+    "@deck.gl/aggregation-layers": "^8.8.0",
```
Comment (Pessimistress) [inline modules/jupyter-widget/package.json:32]: Please never manually change dependency versions. These are managed by lerna. When you publish a production release you need to do so from the *-release branch.
